pkgs/applications/graphics/yed/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper unzip wrapGAppsHook ];
   # For wrapGAppsHook setup hook
-  buildInputs = [ jre.gtk3 ];
+  buildInputs = if stdenv.isDarwin then [ gtk3 ] else [ jre.gtk3 ];
```

Comment:
This will become an anti pattern if everyone would have used a `jre.gtk3` in apps like this. Could you please try and find out if it's possible to add a `passthru` to darwin's jre / jdk (relevant expressions)? I'd much prefer it over a `yEd` specific fix.

Reply:
It's possible, but why add a passthru when it isn't even used by the darwin jre at all? Also, won't that cause a lot of rebuilds by changing the darwin jre?

Reply:
> It's possible, but why add a passthru when it isn't even used by the darwin jre at all?

That's a legitimate question, and I'd argue that it's OK to use it there only for compatibility with the other systems' expressions.

> Also, won't that cause a lot of rebuilds by changing the darwin jre?

No, that's what's nice about `passthru`s - anyone can add them to other people's packages and that won't even trigger the maintainers or trigger rebuilds.

Reply:
Ok, I think that's fair, if other expressions will be using this in a similar way. Good to know about passthru - sounds very useful!